AUTHORS

```diff
@@ -167,5 +167,5 @@ Xavier RENE-CORAIL
 Yehuda B
 Yehuda Bar-Nir
 YFdyh000
-Zach Carter
 Zachary Carter
+Zach Carter
```

Comment:
Why are there changes in this file?

Reply:
Actually I ran the command `yarn run authors` So this happened.